pennylane/devices/tests/test_measurements.py

```diff
@@ -1642,14 +1662,14 @@ def circuit():
             qml.PauliX(0)
             return MyMeasurement(wires=[0]), MyMeasurement(wires=[1])
 
-        with pytest.raises(
-            ValueError, match="Shots must be specified in the device to compute the measurement "
-        ):
+        with pytest.raises(Exception):
```

Comment:
Is there a more specific error type that is expected here?

Reply:
limiting it to `ValueError` and `DeviceError` for now.